ai-platform-unified/notebooks/official/matching_engine/matching_engine_for_indexing.ipynb

```diff
@@ -0,0 +1,1374 @@
+{
```

Comment:
<p>Make sure you change this link.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Thank you!